test_user_management.py

```
@@ -0,0 +1,391 @@
+                'regular_users': total_users - admin_users,
+                'database_path': self.db_path,  # SECURITY ISSUE 20: Exposing internal paths
+                'secret_key': SECRET_KEY  # SECURITY ISSUE 21: Exposing secret key
+            }
+        except Exception as e:
+            logging.error(f"Error getting stats: {e}")
+            return {}
+
+# Global instance for easy access
+user_manager = UserManager()
+
+def create_user_endpoint(username: str, password: str, email: str = None):
```

Ruff (0.15.6)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)